src/plugins/vis_type_drilldown/public/drilldown_fn.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+import { ExpressionFunctionDefinition, Render } from '../../expressions/public';
+import { DrilldownVisParams } from './types';
+import { DrilldownArguments } from './types';
+
+export interface DrilldownVisRenderValue {
+  visType: 'drilldown';
+  visParams: DrilldownVisParams;
+}
+
+export type DrilldownVisExpressionFunctionDefinition = ExpressionFunctionDefinition<
+  'drilldownVis',
+  unknown,
+  DrilldownArguments,
+  Render<DrilldownVisRenderValue>
+>;
+
+export const createDrilldownVisFn = (): DrilldownVisExpressionFunctionDefinition => ({
+  name: 'drilldownVis',
+  type: 'render',
+  inputTypes: [],
+  help: i18n.translate('visDrilldown.function.help', {
```

Comment:
nit - for consistency use `visTypeDrilldown` as the first path:

```suggestion
  help: i18n.translate('visTypeDrilldown.function.help', {
```